13LessonThirteen/homework/css/style.css

```diff
@@ -0,0 +1,100 @@
+header {
+  height: 500px;
+  background-image: url("http://bone.themanoid.com/images/office/03.jpg");
+  text-align: center;
+}
+h1 {
+  line-height: 500px;
+  font-size: 5em;
+  font-family: Work Sans,Helvetica Neue,sans-serif;
+  font-weight: 500;
+}
+.bg-format {
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+#man {
+  background-image: url("http://bone.themanoid.com/images/thumbs/03.png");
+}
+#hills {
+  background-image: url("http://bone.themanoid.com/images/thumbs/07.png");
+}
+#devices {
+  background-image: url("http://bone.themanoid.com/images/thumbs/02.png");
+}
+#rockface {
+  background-image: url("http://bone.themanoid.com/images/thumbs/04.png");
+}
+#building {
+  background-image: url("http://bone.themanoid.com/images/thumbs/05.png");
+}
+#intro {
+  margin-top: 25px;
+}
+.row {
+  margin-bottom: 30px;
+}
+.image-sm{
+  height: 265px;
+}
+aside div {
+  height: 560px;
+}
+#prices {
+  text-align: center;
+  background-color: #f0f0f0;
+  padding-top: 200px;
+}
+#prices h2 {
+  margin-bottom: 100px;
+}
+.items {
+  padding: 30px 0 40px;
+  margin-bottom: 30px;
+  height: 100%;
+}
+.bg-white {
+  background-color: #fff;
+}
+.bg-green {
+  background-color:#d7f7e3;
+  border-bottom: 5px solid #333;
+}
+.btn {
+  background-color: #333;
+  color: #fff;
+  margin-top: 30px;
+}
+.font-bold {
+  font-weight: bold;
+}
+.font-emphasis {
+  font-weight: 200;
+}
+@media screen and (max-width: 768px) {
+  h1 {
+    line-height: normal;
+    font-size: 2.5em;
+    margin-top: 150px;
+    font-family: Work Sans,Helvetica Neue,sans-serif;
+  }
+  #prices {
+    padding: 75px 40px;
+  }
+  #prices h2 {
+    margin-bottom: 50px;
+    font-size: 2.5em;
+    font-family: Roboto Mono,Helvetica Neue,sans-serif;
+    font-weight: 200;
+  }
+  .items h3 {
+    font-size: 1.7em;
+    margin-bottom: 25px;
+  }
+  .items h4 {
+    font-size: 2em;
```

Comment:
try to be consistent with your units of measurement - IMO best practice is to just stick with `px`